docs/topics/feed-exports.rst

```diff
@@ -406,6 +393,10 @@ For instance::
             'postprocessing': [MyPlugin1, 'scrapy.extensions.postprocessing.GzipPlugin'],
             'gzip_compresslevel': 5,
         },
+        'items.jl.xz': {
+            'format': 'jsonlines',
+            'postprocessing': [LZMAPlugin],
+        },
```

Comment:
I think this one can be removed from the example: I don’t think it adds anything, the one above already features a class used directly in `postprocessing`.